docs/guides/admin/docs/configuration/database.md

```diff
@@ -16,28 +16,29 @@ Before following this guide, you should have:
 - Followed the [Basic Configuration instructions](basic.md)
 
 
-Select a Database
------------------
+Step 1: Select a Database
+-------------------------
 
-The EclipseLink JPA implementation which is used in Opencast supports several different databases.
-But some databases might require additional drivers.
+The EclipseLink JPA implementation which is used in Opencast supports several different databases, although
+some databases might require additional drivers.
 Official support only exists for MariaDB, MySQL, PostgreSQL and H2.
 Other database engines are not tested and specific issues will likely not be addressed.
 
 - __MariaDB__ is the recommended database engine.
   It is used by most adopters and is well tested.
 - __MySQL__ is supported but tested less than MariaDB.
-- __PostgreSQL__ support is experimental.
+- __PostgreSQL__ support is experimental. <!-- _Update support here. Is it also less tested than MariaDB -->
```

Comment:
```suggestion
- __PostgreSQL__ support is experimental.
```

Experimental things are not usually well tested yet, are they? :grin:

Reply:
ha ha :P was simply wondering if this was still the case, since things seem to have been outdated in other areas :D